src/schema/tags.ts

```diff
@@ -76,5 +112,47 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
         hits: hits.map((x) => ({ name: x.value })),
       };
     },
+    onboardingTags: async (source, args, ctx): Promise<GQLTagResults> => {
+      const hits = await ctx.getRepository(Keyword).find({
+        select: ['value'],
+        where: {
+          flags: {
+            onboarding: true,
+          },
+        },
+        order: { value: 'ASC' },
+      });
+
+      return {
+        hits: hits.map((hit) => ({ name: hit.value })),
+      };
+    },
+    recommendedTags: async (
+      source,
+      { tags, excludedTags },
+      ctx,
+    ): Promise<GQLTagResults> => {
+      const uniqueTagsToExclude = new Set([...tags, ...excludedTags]);
+
+      if (uniqueTagsToExclude.size > 1000) {
+        throw new ValidationError('Tag limit reached');
+      }
+
+      const hits = await ctx.getRepository(TagRecommendation).find({
```

Comment:
Please add limit here. I think design mentioned 3 or 4 tags

Reply:
Yeah, just checked with product yesterday, it was 5.

Reply:
Added in 053c24c744860b38df07347bfbe8b0f014f2d74a